Hercules-DJControl-Inpulse-200-script.js

```diff
@@ -0,0 +1,218 @@
+// ****************************************************************************
+// * Mixxx mapping script file for the Hercules DJControl Inpulse 200.
+// * Author: DJ Phatso
+// * Version 1.0 (March 2019)
+// * Forum: https://www.mixxx.org/forums/viewtopic.php?f=7&t=12592
+// * Wiki: https://mixxx.org/wiki/doku.php/hercules_djcontrol_inpulse_200
+
+// TODO: Functions that could be implemented to the script:
+//
+// * FX:
+//  - See how to preselect efffects for a rack
+//  - Tweak scratch effect for Pad 4			  
+//
+
+
+
+// ****************************************************************************
+function DJCi200() {}
+
+var DJCi200 = {};
+
+DJCi200.scratchButtonStateDA = true;
+DJCi200.scratchButtonStateDB = true;
+
+
+DJCi200.init = function() {
+
+

```

Comment:
IMHO you went a bit overboard with all that newlines throughout the script, but that's just a preference.